home/index.css

```diff
@@ -220,4 +220,78 @@ nav{
     #home .btn a{
         font-size: 25px;
     }
-}
\ No newline at end of file
+}
+
+/* Start Meals Section */
+#meals{
+    background-image: url("../assets/backGround.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-size: 100% 100%;
+    position: relative;
+    margin-top: 25%;
```

Comment:
If about us section and this PR are merged, white space will appear between them, so the space or margin should delete.

Reply:
Yes, there is a white space between them, we put it.

Reply:
@ahmedirheem which shouldn't